docs/external_contribution_request.md

```diff
@@ -0,0 +1,17 @@
+## Issue title
+
+Publish `EXTENSION_NAME` to Open VSX
+
+## Issue body
+
+```md

```

Comment:
Any reason for wrapping in a code block here? As you don't get the markdown formatting?

Reply:
@loujaybee we decided it would be better for peeps who just go here to copy the template, so they don't copy just the rendered text.

Reply:
I'd then remove all the boilerplate and simply keep the file to copy/paste the full thing? 🤔  

WDYT?